index.js

```diff
@@ -1,4 +1,5 @@
 var bip39 = require('bip39-light')
+var scrypt = require('./vendor/scrypt-async/scrypt-async.min')
```

Comment:
I would keep the non-minified version only and I would move the scrypt-async code to utils folder (you will see it when you rebase to latest master)